### ISSUE TITLE
Add hands-on component

### ISSUE THREAD

**Issue by emdupre** [jbpoline/jb-course-plan#4]
Lessons should have a hands-on component to help participants work through the material. Maybe something like, for Lesson 4, https://try.github.io/.

**Comment by jbpoline** [comment]
yes - actually, the way I am thinking doing this is through several little hands on exercises of say 10' or so.  Thanks a lot for the link and feedback - let me know if you come up with other ideas / comments.
